docs/en/relying-party-solution.rst

```diff
@@ -140,14 +140,36 @@ Below is a non-normative example of the QR Code raw payload:
 
 .. code-block:: text
 
-  ZXVkaXc6Ly9hdXRob3JpemU/Y2xpZW50X2lkPWh0dHBzOi8vdmVyaWZpZXIuZXhhbXBsZS5vcmcmcmVxdWVzdF91cmk9aHR0cHM6Ly92ZXJpZmllci5leGFtcGxlLm9yZy9yZXF1ZXN0X3VyaQ==
+  ZXVkaXc6Ly9hdXRob3JpemU/Y2xpZW50X2lkPWh0dHBzOi8vdmVyaWZpZXIuZXhhbXBsZS5vcmcmcmVxdWVzdF91cmk9aHR0cHM6Ly92ZXJpZmllci5leGFtcGxlLm9yZy9yZXF1ZXN0X3VyaS8=
 
 Below follows its Base64 decoded content:
 
 .. code-block:: text
 
   eudiw://authorize?client_id=https://verifier.example.org&request_uri=https://verifier.example.org/request_uri
 
```

Comment:
I understand that this will be the content of the QR code. If so, how will the RP backend distinguish one request from another?

Reply:
it is a non normative example but anyway we should decide it the reuqest_uri is the same but with urlparams ?state=$that-state or having dynamic request_uri (and then outside of the wallet relying party metadata) that are related to different request_objects (available in the queue)

I think that it would be better taking the JARM recommendation, remove the request_uri from the metadata, and have random request_uri bound to specific state.

@grausof do you agree?